test/models/movie.test.js

```diff
@@ -20,4 +20,60 @@ describe('movie model', () => {
 
   });
 
+  describe('filter', () => {
+    
+    it('filters by exact title', function() {
+      const filter = {
+        title: 'Rollerball'
+      };
+
+      return new Movie().filter(filter).fetchAll()
+      .then(function (movies) {
+        expect(movies.models[0].get('title')).to.eql(filter.title);
+      });
+    });
+
+    it('filters by fuzzy title', function() {
+      const filter = {
+        fuzzy_title: 'and'
+      };
+
+      return new Movie().filter(filter).fetchAll()
+      .then(function (movies) {
+        for (var i = 0; i < movies.models.length; i++) {
```

Comment:
Javascript has a [`.forEach()`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/forEach) method on arrays that allows you to easily iterate through all elements in it. I think we tend to prefer them over for loops as it's somewhat more readable as well as less error prone to make off-by-one mistakes (because you can't 😛).

You can accomplish the same functionality as the for-loop with the following:
```js
movies.forEach((movie) => expect(movie.get('title')).to.contain('and'))
```